builder/upcloud/config.go

```diff
@@ -146,6 +146,12 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
+	if len(c.TemplatePrefix) > 40 {
+		errs = packer.MultiErrorAppend(
+			errs, errors.New("'template_prefix' must be 0-40 characters"),
```

Comment:
1-40 characters, empty currently a template_prefix gets replaced by the default value